server/server.go

```diff
@@ -299,8 +305,8 @@ func CreateServer(ctx context.Context, cfg *config.Config, services []string, le
 		diagnosticspb.RegisterDiagnosticsServer(gs, s)
 		// Register the micro services GRPC service.
 		s.registry.InstallAllGRPCServices(s, gs)
+		s.grpcServer = gs
```

Comment:
When exec `initGRPCServiceLabels`, will it fetch the methods of ETCD gRPC server?

Reply:
Yes, we only have one gRPC server.